test-runner/gh-actions-runner.service

```diff
@@ -0,0 +1,26 @@
+# Install with:
+#   sudo install -m 644 gh-actions-runner.service /etc/systemd/system/
+#   sudo systemctl daemon-reload
+#   sudo systemctl enable gh-actions-runner
+# Run with:
+#   sudo systemctl start gh-actions-runner
+# Stop with:
+#   sudo systemctl stop gh-actions-runner
+# See live logs with:
+#   journalctl -f -u gh-actions-runner.service --no-hostname --no-tail
+[Unit]
+Description=GitHub Actions Runner Container
+After=docker.service
+Requires=docker.service
+[Service]
+TimeoutStartSec=0
```

Comment:
This could lead to the service being stuck in the starting state, is that something we wish?

Reply:
No, definitely seems worth changing. I re-used this setting from the example in https://github.com/myoung34/docker-github-actions-runner/wiki/Usage#systemd. I assume setting `TimeoutStartSec=0` (equivalent to `infinity`) is because the initial docker pull can take some time. Setting it to 5 min seems sufficient for our purposes.